pyuvdata/calfits.py

```diff
@@ -109,11 +129,6 @@ def write_calfits(self, filename, run_check=True, check_extra=True,
             prihdr['FRQRANGE'] = ','.join(map(str, self.freq_range))
         prihdr['TMERANGE'] = ','.join(map(str, self.time_range))
 
-        for p in self.extra():
```

Comment:
So does this mean that we're not supporting arbitrary extra parameters anymore? I think that's okay, just wondering if this was added somewhere else and I missed it.

Reply:
We are, but they go in extra_keywords. This part of the code was apparently to catch actual parameters (meaning ones that are defined explicitly on the object) that were not required to be set and that we didn't want to bother adding to the header explicitly. The problem is that if they need any special handling (like baseline_range) then you have to do them explicitly and then they get caught here as well, which is bad. Also, the keyword in the FITS file gets set by simple truncation which tends to give them somewhat confusing or unclear names and can lead to redundant names. I think this was a bit lazy and caused lots of issues.